capstone/scripts/convert_s3.py

```diff
@@ -76,9 +75,10 @@ def export_cases_to_s3(redacted: bool, reporter_id: str) -> None:
         return
 
     # determine prefix based on a case's frontend_url
-    sample_case_obj = CaseMetadata.objects.select_related().filter(
-        reporter=reporter_id
+    sample_case_obj = list(
+        reporter.case_metadatas.select_related().order_by("case_id")
     )[0]
+
```

Comment:
Can you just use `reporter.short_name_slug` rather than look at cases? I'm not sure how these two techniques compare. You should check in either case if these are unique -- do more than one reporter have the same one? 

If you keep this technique, `list(query)[0]` is less efficient and clear than `query.first()`, which only fetches one row rather than fetching thousands and keeping the first. You also don't want `select_related()` here since you aren't using any related objects on the case.

Reply:
Oof good catch--they're not unique.
These all have 2 reporters with the same slug, which we're using to delineate which reporter is which in our S3 buckets. We could do something similar as we do with the cases and make a 1 and 2 version, but that approach makes less sense to me in this case because it's not that they're actually ordered that way. There might be an argument for using the `analyst_start_year` column to order them, but there are a couple of drawbacks: that column can be null or blank, and some reporters seem to cover the same time period (i.e. one begins in 1941 and goes to 1942, another with the same slug goes from 1941 to 1943). 
Is there a different approach that makes more sense?

The repeats:

```
 short_name_slug | count
-----------------+-------
 bur             |     2
 cal             |     2
 conn-super-ct   |     2
 ct-cl           |     2
 dallam          |     2
 davis-l-ct-cas  |     2
 haw             |     2
 ind-app         |     2
 la              |     2
 lack-bar-r      |     2
 leg-gaz         |     2
 luz-leg-reg     |     2
 mass-app-div    |     2
 nys-2d          |     2
 va              |     2
```

Reply:
Can you take a look at each, like https://cite.case.law/bur/? My guess is in most cases one or both of these doesn't have volumes, and if we exclude those this goes away. The exception I saw in the first few was Ct. Cl., where one is West Virginia Court of Claims and one is US Court of Claims but both reporters properly have their cases cited as "1 Ct. Cl. 1" -- https://cite.case.law/ct-cl/1/. SO ... us-ct-cl and wv-ct-cl would make sense for the folder names, but I'm not sure if it would break anything to set reporter short name that way in the database or if we need a special case in the export script.

Reply:
Proposal: new naming convention as slug-reporter_id (i.e. `ind-app-436` and `ind-app-698`)

Reply:
Ticket created here: https://linear.app/harvard-lil/issue/ENG-283/determine-how-to-differentiate-among-reporters-with-same-slug